android/assets/jsons/Nations/Nations_Ukrainian.json

```diff
@@ -236,7 +236,7 @@
 		"name": "Rome",
     "translatedName": "Рим",
 		"leaderName": "Октавіан Август",
-		"adjective": ["Roman"],
+		"adjective": ["Римський"],
```

Comment:
Why some translations are `Римський`, `Китайський` and another `Російська`, `Французька` ? Is the gender random?

Reply:
In Ukrainian, you can say "`Римське місто (Roman city)`", here Rome is used as a neuter, but you can also say "`Римський правитель (Roman leader)`", here Rome is used as a masculine, so it is impossible to pick up one adjective. I tried to find one that would fit the initial form of the state name: `Рим (Rome)` (on ukrainian it is a male), `Франція (France)`(on ukrainian it is a female)

Reply:
Thank you for an explanation. In this context it makes sense. I did not find in the game how it is used so far. Let's see how it will fit in the game actually.